components/molecule/accordion/src/index.js

```diff
@@ -31,17 +31,21 @@ const MoleculeAccordion = ({
 
   return (
     <div className={BASE_CLASS}>
-      {children.map((child, index) => (
-        <Tab
-          isOpen={!!openTabs[index]}
-          key={index}
-          onToggle={_handleOnToggle(index)}
-          title={child.props.label}
-          {...tabProps}
-        >
-          {child.props.children}
-        </Tab>
-      ))}
+      {children.map((child, index) => {
+        const isDefaultOpenedTab = initialOpenTabs[index]
+        const isCurrentOpenedTab = openTabs[index]
+        return (
+          <Tab
+            isOpen={isDefaultOpenedTab || isCurrentOpenedTab}
```

Comment:
In my opinion this feature makes imposible to close an initially opened tab. Isn't it?